Gems/Atom/Feature/Common/Assets/Shaders/SkyAtmosphere/SkyVolumeLUT.azsl

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+ // Based on https://github.com/sebh/UnrealEngineSkyAtmosphere by Sébastien Hillaire
+
+#define ENABLE_ATMOSPHERE_SHADOWS 0
+#include <viewsrg.srgi>
+#include <Atom/Features/SrgSemantics.azsli>
+#include <SkyAtmosphereCommon.azsli>
+
+float4 GetSkyVolumeAtPosition(uint3 position)
+{
+    AtmosphereParameters Atmosphere = GetAtmosphereParameters();
+
+    const float worldScale = 0.001f;
+    const float lutSize = float(SKY_VOLUME_SLICE_COUNT - 1);
+    float2 uv = float2(position.x, position.y) / lutSize;
+    float2 ndcPos = float2(uv.x , 1.0f - uv.y) * 2.0f - 1.0f;
+    float4 projectedPos = float4(ndcPos, 0.f, 1.0f);
+    float4 positionWS = mul(ViewSrg::m_viewProjectionInverseMatrix, projectedPos);
+    positionWS /= positionWS.w;
+
+    float3 viewToPosition =  positionWS.xyz / positionWS.w - ViewSrg::m_worldPosition;
```

Comment:
Did you intend on doing the perspective divide here a second time?

Reply:
Is this a typo?